tock/hours/views.py

```diff
@@ -613,6 +616,71 @@ def get_object(self, queryset=None):
             user_id=self.request.user.id)
         return obj
 
+    def clean_task_data(self, float_people_id):
+        # Get Float task data for week corresponding w/ ReportingPeriod
+        # instance.
+        r = get_float_data(
+            endpoint='tasks',
+            params={
+                'weeks': base.FLOAT_API_TASK_WEEKS,
+                'start_day': datetime.datetime.strptime(
+                    self.kwargs['reporting_period'], "%Y-%m-%d"
+                ).date()
+            }
+        )
+        if r.status_code != 200:
+            return None
```

Comment:
Cool--the "django-y" way to do it is via [the Django/Python logging system](https://docs.djangoproject.com/en/1.11/topics/logging/) but honestly I find that to be one of the most confusing things I've ever encountered in programming, so it's up to you.  `print` is fine with me as long as it's okay with whoever is doing devops for Tock!